sycl/test/basic_tests/accessor/accessor_property_list.cpp

```diff
@@ -0,0 +1,111 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
Can we split test in two part: compile time and compile+run time.
Do we really need to run these tests as accessor properties are compile-time?

Reply:
Split run time and compile time tests, run time tests now only run on host, as they're device-agnostic.